### ISSUE TITLE
A suggestion on governance changes

### ISSUE THREAD

**Issue by atheriel** [cranchange/cran_change.org#26]
One thing I don't see mentioned in the letter but is tied up with a call for improved governance: in my view, CRAN should establish a formal channel by which community members can *become* CRAN administrators -- perhaps with a shadow program or mentorship obligation.

**Comment by MilesMcBain** [comment]
Yeah. Good point. By contrast, the R foundation lays out this out very clearly in its charter.

I'll try to include this in the petition. Do you have any examples I could point to?

**Comment by atheriel** [comment]
I'm very inspired by the success of the shadow program for [Kubernetes releases](https://github.com/kubernetes/sig-release/blob/master/release-team/shadows.md), which has churned out dozens of new highly-informed release team members in the last few years.

**Comment by MilesMcBain** [comment]
Wow that example is very inspiring, thankyou!

**Comment by atheriel** [comment]
@MilesMcBain Yes, I heard a couple interviews with Kubernetes release leads and it really made me realize how much better governance could be in the R ecosystem.

**Comment by wlandau** [comment]
On reflection, I think governance precedes most of the 11 service-related implementation-level reforms proposed in the letter. CRAN is not run sustainably. If it continues to be managed by a small number of volunteers long-term, some of whom are emeritus, we face severe problems a couple of decades down the road. The incidents we see now are likely to be symptoms. But in a way, governance is the most sensitive issue of all, and I do not see the discussion going well if we directly petition CRAN about it. So as I mention in https://github.com/cranchange/cran_change.org/issues/27#issuecomment-877500670, it might be best to initiate dialogue with different folks in equally high places.

**Comment by atheriel** [comment]
@wlandau I too think that direct discussion is a more likely route to change. But I also think starting those discussions with a very concrete set of goals and proposals in mind is a good idea, and that's one thing this kind of format provides.
